src/main/java/com/basilisk/frontend/views/HomePageView.java

```diff
@@ -0,0 +1,100 @@
+package com.basilisk.frontend.views;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.tabs.Tab;
+import com.vaadin.flow.component.tabs.Tabs;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.router.Route;
+
+@Route("Home")
+public class HomePageView extends VerticalLayout {
+    Integer textAreaCounter = 10;
+    String hexTurquoise = "#40E0D0";
```

Comment:
Integer textAreaCounter
String hexTurquoise

Should be Private and if they never change then they should be final